docs/content.zh/docs/ops/metrics.md

```diff
@@ -1277,7 +1277,7 @@ Note that the metrics are only available via reporters.
   </thead>
   <tbody>
     <tr>
-      <th rowspan="20"><strong>Job (only available on TaskManager)</strong></th>
+      <th rowspan="8"><strong>Job (only available on TaskManager)</strong></th>
```

Comment:
This change should probably be replicated in the Chinese version.

Reply:
Which you pointed out is just chinese version ? We are in content.zh currently ?

Reply:
Sorry, I meant to synchronize the both versions (in regard to this edit).